modules/admin_manual/pages/installation/manual_installation/manual_imagick7.adoc

```diff
@@ -195,9 +225,31 @@ sudo service php7.4-fpm restart
 +
 [source,console]
 ----
-php -r 'phpinfo();' | grep ImageMagick
+php -r 'phpinfo();' | grep -i "ImageMagick supported formats"
+----
+
+== TIPS
+
+=== Reinstall or upgrade ImageMagick-7 and the php wrapper.
+
+To reinstall or upgrade ImageMagick-7, follow the principle steps described above by disabling and removing the php wrapper first, rerun the imei.sh installation script with the options of choice. The script checks if components need an upgrade and installs them in case. Then reinstall the php wrapper, enable it and restart your webserver or php-fpm.
+
+=== Uninstall ImageMagic-7 or components
+
+If you want to uninstall ImageMagick-7 only, run:
+
+[source,console]
+----
+sudo apt remove imei-imagemagick
+----
+
+If you want to completely remove ImageMagic-7 and all of its installed components, run:
+
+[source,console]
+----
+sudo apt remove imei-imagemagick,imei-libaom,imei-libheif,imei-libjxl
 ----
 
-== TIP
+=== Change Configuration Settings of ImageMagic-7
 
-If you want to install a new version of ImageMagick-7 and/or php wrapper, repeat all steps from the beginning.
+You can change configuration settings of ImageMagick-7 at any time accoring your needs. In case you do so, restart your webserver of the php-fpm service post changing the settings so they can take effect for web services.
```

Comment:
```suggestion
You can change configuration settings of ImageMagick-7 at any time according to your needs. In case you do so, restart your web server of the php-fpm service post changing the settings so they can take effect for web services.
```